src/librustc_session/session.rs

```diff
@@ -1349,6 +1349,9 @@ fn validate_commandline_args_with_session_available(sess: &Session) {
         && sess.panic_strategy() == PanicStrategy::Unwind
         && sess.opts.prints.iter().all(|&p| p == PrintRequest::NativeStaticLibs)
     {
+        // FIXME(richkadel): Discover if tcx.sess.opts.debugging_opts.instrument_coverage
+        // might also "not yet work in conjuction with `-Cpanic=unwind` on Windows when targeting
+        // MSVC". If so, add it to the `if` condition above.
```

Comment:
Flagging this so we don't forget

Reply:
I'm actually going to remove this comment. The more I think about it, the more I'm convinced this is a PGO problem and unrelated to InstrProf. I already removed the parallel comment and check from the `run-make-fulldeps/instrument-coverage` test, BTW.